app/assets/javascripts/discourse/tests/setup-tests.js

```diff
@@ -298,15 +297,15 @@ function setupTestsCommon(application, container, config) {
 
     createHelperContext({
       get siteSettings() {
-        if (isLegacyEmber() && container.isDestroyed) {
```

Comment:
Under modern ember this was always `true`, but now it  depends on `isDestroyed`. I think we can remove this branch of the if statement.